src/main/resources/templates/admin/scheduling.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Scheduling</title>
+    <div th:replace="~{common :: styling}"></div>
+</head>
+<body>
+<header>
+    <div th:replace="~{common :: navbar}"></div>
+</header>
+
+<h2 th:if="${error}" class="error" th:text="${'There is already a question scheduled on that day'}"></h2>
```

Comment:
Good use of the th:if thymeleaf, though you could probably have just put the message in regular text instead of using th:text.